### PR TITLE
Update implementation notes for R4 Condition create

### PR DESCRIPTION
**Pull request by MaxPhilips**
Description
----
Adds a new R4 Condition create immplementation note:

```
An `encounter-diagnosis` Condition will be automatically prioritized to the least significant priority by the Cerner Millennium EHR
```

and rewords some of the others for clarity

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by MaxPhilips** [comment]
local verification

Screen Shot 2021-03-29 at 1 35 09 PM

**Review by kathypickering — APPROVED** [review]

File: content/millennium/r4/general-clinical/condition.md [hunk 1]
```diff
@@ -164,10 +164,13 @@ Create a new Condition.
...
-* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* The Condition Create API supports only the API fields mentioned below. Unsupported fields will be ignored.
 * Modifier fields should not be provided, and will cause the transaction to fail.
-* The code.coding field can have at most 2 codings, one of which must be set as userSelected true and the other one must be set as userSelected false.
-* Currently `problem-list-item` and `encounter-diagnosis` are supported.
+* The code.coding field can have at most 2 codings.
+  * One of the codings must have userSelected: true
+  * The other coding must have userSelected: false
+* The Condition Create API currently supports only Conditions with a category of `problem-list-item` or `encounter-diagnosis`.
+* An `encounter-diagnosis` Condition will be automatically prioritized to the least significant priority by the Cerner Millennium EHR.
```
Comment (sydneanschutz) [inline content/millennium/r4/general-clinical/condition.md:173]: Are we updating this for DSTU2 as well? AC: Update fhir.cerner.com R4 and DSTU2 Condition Create Implementation Notes by adding...

Comment (kathypickering) [reply content/millennium/r4/general-clinical/condition.md:173]: Yes, we should update DSTU2 as well

Comment (MaxPhilips) [reply content/millennium/r4/general-clinical/condition.md:173]: https://github.com/cerner/fhir.cerner.com/pull/602/commits/f6310eebfe043f0f9b9f9eb43cff9c65ca7a9d4a

**Comment by MaxPhilips** [comment]
dstu 2 local verification

Screen Shot 2021-03-29 at 2 07 15 PM

**Review by sydneanschutz — APPROVED** [review]

**Comment by Anurag-Balannagari** [comment]
verified r4 condition changes in live
image

**Comment by Anurag-Balannagari** [comment]
verified dstu 2 condition changes in live
image

**Pull request merged**